src/Libraries/Microsoft.Extensions.AI.Evaluation.NLP/Directory.Build.targets

```diff
@@ -0,0 +1,33 @@
+<Project>
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)..\'))" />
+
+  <!-- Generate a version constant that can be accessed from the C# code. -->
+  <Target Name="GenerateVersionConstant" BeforeTargets="CoreCompile">
+    <PropertyGroup>
+      <_ConstantsFilePath>$(IntermediateOutputPath)Constants.g.cs</_ConstantsFilePath>
+      <_Lines>
+// <auto-generated>
+// This file is auto-generated by MSBuild.
+// </auto-generated>
+
+namespace $(RootNamespace)%3B
+
+internal static class Constants
+{
+    public const string Version = "$(Version)"%3B
```

Comment:
Is `Version` being used in the code in this project or was this file just created as a result of a copy paste? Please delete the file if the const is unsed in the code.

Reply:
All of our libraries include that version number type. Seems like it would be a good idea to retain consistency here.